newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -1083,7 +1092,10 @@ export default class AuthenticatedUserProvider extends React.Component<
         // by the API when fetched.
       }
 
-      await this._fetchUserProfileWithoutThrowingErrors();
+      await this._fetchUserProfileWithoutThrowingErrors({
+        // When creating an account, avoid showing the email verification dialog right away.
+        dontNotifyAboutEmailVerification: true,
```

Comment:
I think this is fine, because most users use a SignUpProvider now + when opening the Profile Dialog that popup will then show (or next time the app is launched)

Reply:
Ok 👍